packages/x-adapter-platform/src/types/models/related-prompt.model.ts

```diff
@@ -7,4 +9,26 @@ export interface PlatformRelatedPrompt {
   nextQueries: string[];
   suggestionText: string;
   type: 'SYNTHETIC' | 'CURATED';
+  tagging: PlatformRelatedPromptTagging;
```

Comment:
```ts
export interface Taggable {
  /** Tagging object containing the different taggable events. */
  tagging?: Tagging;
}
```
Are we "deprecating" this interface?  (IMO, is an interface with all optional, for me.. I prefer your approach, just asking)